src/integrationTest/resources/addaudio/connection.properties

```diff
@@ -1,4 +1,4 @@
 moduleName = darts
 repository = moj_darts
-user = dmadmin
-password = Hym5Aro7/irTfKQgMSdvow==
+user = viq_ws_user
+password = Amu49glKfaxTcKueW+H2dA==
```

Comment:
Should these be secrets?

Reply:
Documentum client library requires the user/password to be read from a local file where the password is encrypted

Reply:
Should these be loaded in via secrets instead?

Reply:
This has been fixed. We have eliminated the storage of the password